docs/pages/blog/mui-x-end-v8.md

```diff
@@ -0,0 +1,251 @@
+  <figcaption style="text-align:center;margin-top:8px;">Keyboard-first exploration of series and points.</figcaption>
+</figure>
+
+[Accessibility and keyboard](/x/react-charts/accessibility/)
+
+### Sankey and upgraded charts [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+A **Sankey** diagram joins the lineup for flow visualization, while **Funnel** charts gain polished presets (including pyramid/step-pyramid) so conversion stories read at a glance.
+
+<figure style="margin:16px 0;">
+  <img src="/static/blog/mui-x-end-v8/charts-sankey-funnel.png" alt="Sankey and Funnel charts" style="width:100%;max-width:960px;border-radius:12px;">
+  <figcaption style="text-align:center;margin-top:8px;">Tell flow and conversion stories clearly.</figcaption>
+</figure>
+
+[Sankey docs](/x/react-charts/sankey/) • [Funnel docs](/x/react-charts/funnel/)
+
+### Clearer axes for dense categories
+
+Multi-level **label groups** on band/point axes tidy up long category lists and comparisons — perfect for “Region Country City” break-downs.
+
+<figure style="margin:16px 0;">
+  <img src="/static/blog/mui-x-end-v8/charts-axis-groups.png" alt="Grouped axis labels" style="width:100%;max-width:960px;border-radius:12px;">
+  <figcaption style="text-align:center;margin-top:8px;">Multi-level category labels.</figcaption>
+</figure>
+
+[Axes and labeling](/x/react-charts/axes/)
+
+### Performance and export [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+Heavy dashboards feel lighter thanks to faster string measurement, tuned bar rendering, and an **SVG batch renderer** for large scatter plots. When it’s time to share, capture the exact view with **image export** or **print/PDF**.
+
+<figure style="margin:16px 0;">
+  <img src="/static/blog/mui-x-end-v8/charts-perf-export.png" alt="Charts performance and export" style="width:100%;max-width:960px;border-radius:12px;">
+  <figcaption style="text-align:center;margin-top:8px;">Faster charts; one-click export.</figcaption>
+</figure>
+
+[Performance tips](/x/react-charts/scatter/#performance) • [Export guide](/x/react-charts/export/)
+
+## Data Grid
+
+### Server-side pivoting [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+Pivot at scale without locking the browser. The Grid coordinates with your backend to compute pivots and aggregates, so the UI stays snappy while you slice by Region Segment Product (or any model you dream up).
+
+<figure style="margin:16px 0;">
+  <video src="/static/blog/mui-x-end-v8/grid-pivot.mp4" autoplay muted loop playsinline controls style="width:100%;max-width:960px;border-radius:12px;"></video>
+  <figcaption style="text-align:center;margin-top:8px;">Large pivots, computed on the server.</figcaption>
+</figure>
+
+[Start with pivoting now](/x/react-data-grid/pivoting/).
```

Comment:
some of these links appear to be full sentences, others are incomplete sentences, and still others are just names. Some have periods and others don't. Can we standardize them with one style across the board?